UxScenarios/Scenario1_answer/Program.cs

```diff
@@ -26,20 +27,22 @@ static void Main(string[] args)
             // }
         }
 
-        // TODO:
-        // 1) Use JsonSerializer to serialize the "account" object to a JSON string and return it.
-        // 2) Then, modify the code to "pretty-print" the JSON so it is indented, using JsonSerializerOptions.
+        // TODO: 1) Use JsonSerializer to serialize the "account" object to a JSON string and return it.
         private static string Serialize(Account account)
         {
-            // 1a) Find the right API overload to call, with the correct signature
-            // string jsonString = JsonSerializer.ToString<Account>(account);
-            // return jsonString;
+            // Find the right API overload to call, with the correct signature
```

Comment:
nit: can you keep the "numbered" steps (to be consistent with the other answers)

So, this would be 1a)

```suggestion
            // 1a) Find the right API overload to call, with the correct signature
```

Reply:
We can change the structure of all of them based on feedback, but I would try to remain consistent.